uobtheatre/productions/migrations/0027_alter_production_options.py

```diff
@@ -0,0 +1,29 @@
+# give users with change_production permission the apply_booking_discount permission
+# god save us all this might break everything
+
+from django.db import migrations
+from django.db import migrations, models
+
+
+def add_apply_booking_discount(apps, schema_editor):
+    Permission = apps.get_model('auth', 'Permission')
+    Group = apps.get_model('auth', 'Group')
+    apply_booking_discount = Permission.objects.get(codename='apply_booking_discount')
+    change_production = Permission.objects.get(codename='change_production')
+    for group in Group.objects.filter(permissions=change_production):
+        group.permissions.add(apply_booking_discount)
```

Comment:
Should also find users who have the permission, not just groups? Apart from that I think this seems to do what it should do, but we should do a quick run locally to make sure